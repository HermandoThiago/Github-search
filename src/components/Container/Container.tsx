export default function Container(){
    return (
        <div></div>
    );
};