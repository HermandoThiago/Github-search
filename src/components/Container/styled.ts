import styled from "styled-components";

export const StyledContainer = styled.section`
    width: 100%;
    max-width: 1280px;
    margin: 0 auto;
    padding: 0 20px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
`;
