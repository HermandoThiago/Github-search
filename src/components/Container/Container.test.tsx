import renderer from 'react-test-renderer';
import Container from './Container';

describe('Tests Container', () => {
    it('should match snapshot', () => {
        const component = renderer.create(
            <Container>
                <p>container</p>
            </Container>
        ).toJSON();

        expect(component).toMatchSnapshot();
    })
})