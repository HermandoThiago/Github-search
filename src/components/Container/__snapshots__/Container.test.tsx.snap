// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Container should match snapshot 1`] = `
<section
  className="sc-bczRLJ kapKWY"
>
  <p>
    container
  </p>
</section>
`;
